pkg/operator/ceph/disruption/clusterdisruption/mon.go

```diff
@@ -61,9 +64,29 @@ func (r *ReconcileClusterDisruption) reconcileMonPDB(cephCluster *cephv1.CephClu
 			MinAvailable: &intstr.IntOrString{IntVal: minAvailable},
 		},
 	}
-	err := r.reconcileStaticPDB(pdbRequest, pdb)
-	if err != nil {
-		return errors.Wrap(err, "could not reconcile mon pdb")
+
+	existingPDB := &policyv1beta1.PodDisruptionBudget{}
+	err := r.client.Get(context.TODO(), pdbRequest, existingPDB)
+	if apierrors.IsNotFound(err) {
```

Comment:
How about this pattern?
```
if err != nil {
  if apierrors.IsNotFound(err) {
    // insert lines 71-75 to create the pdb
  }
  return errors.Wrap(err, "failed to get mon pdb")
}
```

Reply:
Yes, this looks like the common pattern used across rook. Changed code to use this format.